winerp/client.py

```diff
@@ -110,6 +110,26 @@ async def __verify_client(self):
         await self.send_message(payload)
         logger.info("Verification request sent")
 
+    async def clients(self, timeout: int = 60):
+        _uuid = str(uuid.uuid4())
+        payload = MessagePayload(
+            type=Payloads.response,
+            id=self.local_name,
+            uuid=_uuid,
+            destination="server"
```

Comment:
Not good to name it as `server`, as someone can have a client with that name.
kindly change it to `winerp_server` instead.

Reply:
Also would be good to not let people name their client as `winerp_server` when registering a client.